### ISSUE TITLE
github content not correct

### ISSUE THREAD

**Issue by jazel1969** [PacktPublishing/REST-APIs-with-Flask-and-Python#1]
I am following REST-APIs-with-Flask-and-Python video training.
It looks like the content stored on github are for different training. 
It contains java files while python is expected.

**Comment by 51naa** [comment]
> I am following REST-APIs-with-Flask-and-Python video training. It looks like the content stored on github are for different training. It contains java files while python is expected.

You're right. Try [this ](https://github.com/tecladocode/rest-apis-flask-python) instead.
